Source/WebCore/accessibility/AccessibilitySVGObject.cpp

```diff
@@ -319,4 +327,59 @@ AccessibilityRole AccessibilitySVGObject::determineAccessibilityRole()
     return AccessibilityRenderObject::determineAccessibilityRole();
 }
 
+void AccessibilitySVGObject::setParent(AccessibilityRenderObject* parent)
+{
+    m_parent = parent;
+}
```

Comment:
Should we `ASSERT(isRoot)`? This shouldn't happen for non-root things, right? Also, maybe worth defining this short method in the header to help the compiler inline the call?